inductiva/stellarators/scenarios/coils/coils.py

```diff
@@ -141,6 +141,59 @@ def from_random_curves(cls,
 
         return cls(coils, num_field_periods)
 
+    @classmethod
+    def from_curves_file(cls,
+                         num_field_periods,
+                         num_coils,
+                         coil_currents,
+                         curves_file,
+                         delimiter=',',
+                         max_order=14):
+        """Create StellaratorCoils from Fourier coefficients loaded from a file.
+
+        This function loads a file containing Fourier coefficients for several 
+        coils. The file is expected to have `6*num_coils` many columns, and 
+        `order+1` many rows. The columns are in the following order,
+
+        sin_x_coil1,cos_x_coil1,sin_y_coil1,...,sin_x_coil2,cos_x_coil2,...
```

Comment:
Use a consistent notation. You have previously called this s_j, sj, etc. Make it consistent everywhere you mention them in this class.